python/pyspark/ml/tests/connect/test_connect_function.py

```diff
@@ -0,0 +1,113 @@
+#
```

Comment:
I think you can remove this test - I believe the doctests in `pyspark.ml.connect.functions` cover the same cases.